plugins/CoreAdminHome/OptOutManager.php

```diff
@@ -248,13 +613,13 @@ private function optOutStyling()
             }
         }
 
-        if ($cssfontsize && (preg_match("/^[0-9]+[\.]?[0-9]*(px|pt|em|rem|%)$/", $cssfontsize))) {
+        if ($cssfontsize && (preg_match("/^[0-9]+[.]?[0-9]*(px|pt|em|rem|%)$/", $cssfontsize))) {
```

Comment:
doesn't removing the `\` mean, that instead of an optional `.`, now any character can be there? 🤔

Reply:
As above: `@noinspection RegExpRedundantEscape` added.